src/RemoteInteractor.ts

```diff
@@ -0,0 +1,65 @@
+import * as http from "http";
+import * as https from "https";
+import * as URL from "url";
+import {InteractionModel} from "./InteractionModel";
+import {ActionInteractor} from "./ActionInteractor";
+
+/**
+ * ActionInteractor works with an action via HTTP calls to a URL
+ *
+ */
+export class RemoteInteractor extends ActionInteractor{
```

Comment:
Looks like this part is not tested

Reply:
This is tested by the old tests, Remote Interactor invoke was the method used on ActionInteractor, now that action interactorm is an abstract method, the invoke is implemented and called here.